public/app/features/api-keys/ApiKeysPage.tsx

```diff
@@ -116,6 +116,11 @@ export class ApiKeysPageUnconnected extends PureComponent<Props, State> {
     }
   };
 
+  //@TODO Remove this, it's only for dev testing
+  produceErrorr = () => {
+    throw new Error('This is a test error');
```

Comment:
This obviously has to go before merge, left it here to enable local testing for whoever is reviewing.

Reply:
The failing a11y tests will be green again after we remove this